Addresses some fixes/additions

0x5066 opened the pull request:
This commit fixes the wrong character bitmap which was missing a \.

This also tries to get the "status" object working for playback, yet doesn't render.

This also attempts to get "getStatus()" working, though untested.

0x5066 committed:
Addresses some fixes/additions

This commit fixes the wrong character bitmap which was missing a \.

This also tries to get the "status" object working for playback, yet doesn't render.

This also attempts to get "getStatus()" working, though untested.

captbaritone commented on packages/webamp-modern-2/src/skin/makiClasses/Status.ts:
```diff
@@ -1,17 +1,74 @@
       return true;
     }
     switch (key) {
+      case "stopBitmap":
```
I think `key` should be lower case here (see line 12), so we probably want these to be lower case as well.

0x5066 replied:
good catch! just got that working

0x5066 committed:
fixed the status object not rendering

still doesn't address the issue of it not being aware of the player status, but it's a start!

x2nie reviewed (approved):
looks good
